core/trino-main/src/main/java/io/trino/operator/AggregationMetrics.java

```diff
@@ -16,18 +16,40 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.collect.ImmutableMap;
+import io.airlift.units.Duration;
+import io.trino.plugin.base.metrics.DurationTiming;
 import io.trino.plugin.base.metrics.LongCount;
 import io.trino.spi.metrics.Metric;
 import io.trino.spi.metrics.Metrics;
 
+import static java.util.concurrent.TimeUnit.NANOSECONDS;
+
 public class AggregationMetrics
 {
     @VisibleForTesting
     static final String INPUT_ROWS_WITH_PARTIAL_AGGREGATION_DISABLED_METRIC_NAME = "Input rows processed without partial aggregation enabled";
+    private static final String ACCUMULATOR_TIME_METRIC_NAME = "Accumulator update wall time";
+    private static final String GROUP_BY_HASH_TIME_METRIC_NAME = "Group by hash update wall time";
```

Comment:
I was considering this. Both Accumulators and GroupByHashes operate on `Pages`/`Blocks`. I assumed we cannot guarantee that a `Block` is loaded in memory, meaning accessing it could generate I/O calls. WDYT?

Reply:
> I assumed we cannot guarantee that a Block is loaded in memory

Aggregation operator will always have blocks fully loaded. Only ScanFilterProjection could load blocks lazy